data/pages.yaml

```diff
@@ -53,6 +49,56 @@ novice:
   learn: Understand the fundamentals of Ansible automation
   install: Install the Ansible package
   start: Write your first playbook in a few easy steps
+# Resources for user journeys
+user:
+  title: User resources
+  intro: Automate the management of remote systems and control their desired state.
+  milestone:
+    automate: Create automation
+    build_inventory: Build inventories
+    organize: Organize automation projects
+    tooling: Use Ansible tooling
+    reuse: Re-use automation content
+    share: Share automation content
+    awx: Schedule and run automation jobs
+    executor: Embed automation in systems
+  automate:
+    intro: Write playbooks and develop content that automates complex tasks.
+    start: Start writing Ansible playbooks
+    evaluate_plugins: Learn which plugins to use for your tasks
+  build_inventory:
+    intro: Build inventories that list and describe your infrastructure.
+    multiple_hosts: Build inventory files to manage multiple hosts
+    dynamic: Use dynamic inventories
+  organize:
+    intro: Organize your automation project as it grows.
+    roles: Use roles to structure the automation project
```

Comment:
more details on what I mean by only using this page to point to other projects. I'd remove automate, build_inventory, and organize. They are covered on the front page.

Reply:
@samccann agreed. thanks for the review comment here.

Reply:
https://github.com/ansible/jinja-docsite/issues/44